gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
I wish that the new framework is completely multi-platform, so I would like to have the `gradlew.bat` also tracked in the repository. Lately, we can setup an [AppVeyor](https://www.appveyor.com/) run for testing the build on Windows. This will be useful for check if we are correctly setting up paths (I don't want hardcoded path separators, but rather use filesystem methods to generate them).

Reply:
That's fine with me as long as you are willing to be the champion of the windows builds.  I have essentially no interest in supporting windows but I have no objections.

Reply:
Using java.nio and its method for path construction/resolution should be enough to support windows, and I am willing to check the builds and test them in my computer. I would like to have a complete multi-platform support to allow development and usability of the library in different contexts (e.g., multi-platform GUIs like IGV or command line java tools).

But anyway, I can setup the builds with AppVeyor and the gradle wrapper once this is in.